### ISSUE TITLE
Violation of design

### ISSUE THREAD

**Issue by webmastak** [johnfactotum/gedit-restore-minimap#3]
With plugin restore-minimap enabled:
![01](https://i.imgur.com/Whew3Ni.png)

If the plugin restore-minimap is disabled:
![02](https://i.imgur.com/5JguBjX.png)

**Comment by johnfactotum** [comment]
Yes, this is [a known issue](https://github.com/johnfactotum/gedit-restore-minimap#known-issues): "It breaks when an info bar is displayed".

gedit does not support arbitrary panels around the editor, and the current method used by this plugin is a hack, which actually works okay apart from this issue.

**Comment by webmastak** [comment]
You can make it so that when an info bar is displayed so that the minimap is hide or the info bar always be displayed on top?


Another minor mistake:
![03](https://i.imgur.com/IpniREW.png)

I also noticed that it does not always break when displaying an info bar:
![04](https://i.imgur.com/6xTYzsj.png)
Can the info bar always be displayed on top?

**Comment by ahmubashshir** [comment]
I tried to fix this with #5, but it breaks snippets... I'll try to address that.
